tonic/models/vic/grid_params.py

```diff
@@ -632,6 +845,9 @@ def grid_params(soil_dict, target_grid, snow_dict, veg_dict, veglib_dict,
             if mydict[var].dtype in [np.int, np.int64, np.int32]:
                 fill_val = FILLVALUE_I
                 dtype = np.int
+            elif mydict[var].dtype in [np.str]:
```

Comment:
I think the most Pythonic way of doing this would be to use the `isinstance` function. For example:

```Python
elif isinstance(mydict[var].dtype, np.str):
    ....
```